### PR TITLE
Include default vars to filebeat's upgrade role

### PR DESCRIPTION
**Pull request by rpudlowski93**
- PR releated to failed test in task #1833 
- Default vars included to filebeat upgrade role from filebeat role
- Filebeat chart name moved from specification to defaults

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/filebeat.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---
+- name: Filebeat | Load defaults from filebat role
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/filebeat.yml:2]:
```suggestion
- name: Filebeat | Load defaults from filebeat role
```

**Review by to-bar — CHANGES_REQUESTED** [review]

**Review by to-bar — APPROVED** [review]

**Comment by przemyslavic** [comment]
@rpudlowski93 I found a bug while testing fix for failing upgrades [#1875](https://github.com/epiphany-platform/epiphany/issues/1875) (PR [#1876](https://github.com/epiphany-platform/epiphany/pull/1876)). I deployed a cluster without kubernetes and helm. During upgrade it fails on task `Get information about helm releases`. The entire block `Filebeat as DaemonSet | Prepare fact about helm release` should be run conditionally (when k8s_as_cloud_service). Same for node exporter.

**Review by to-bar — APPROVED** [review]

**Review by atsikham — APPROVED** [review]

**Pull request merged**